Islandora-CLAW/controlled_access_terms
# part of a fix for https://github.com/Islandora-CLAW/CLAW/issues/1092

making taxonomy term fields translatable in the default configuration

**GitHub Issue**: (https://github.com/Islandora-CLAW/CLAW/issues/1092)

# What does this Pull Request do?
Make the taxonomy and taxonomy term fields translatable with the default configuration

# What's new?
Add yml configuration for language.content_settings.taxonomy_term.[taxonomy_name].yml for corporate_body, family, geo_location, person, resource_types, and subject taxonomies. Add the translatable:true for the following taxonomy term fields in the field configuration: corporate_body.field_cat_date_begin, corporate_body.field_corp_alt_name, corporate_body.field_type, family.field_authority_link, family.field_cat_date_begin, geo_location.field_authority_link, geo_location.field_geo_alt_name, geo_location.field_geo_broader, geo_location.field_geo_geolocation, person.field_authority_link, person.field_cat_date_begin, person.field_cat_date_end, person.field_person_alternate_names, person.field_person_preferred_name, person.field_relationships, resource_types.field_external_authority_link, subject.field_authority_link

# How should this be tested?
Copy down the updated config, import controlled_access_terms_default_configuration (ie. `drush fim controlled_access_terms_default_configuration`). Check to make sure you can translate the terms and fields: http://localhost:8000/admin/config/regional/content-language show demonstrate the taxonomies and fields are checked and able to be translated.

# Interested parties
@Natkeeran
